lib/v2/components/text_form_field_custom.dart

```diff
@@ -34,10 +34,10 @@ class TextFormFieldCustom extends StatelessWidget {
     this.keyboardType = TextInputType.text,
     this.controller,
     this.onChanged,
-    this.textCapitalization,
+    this.textCapitalization = TextCapitalization.none,
```

Comment:
what if we need this to have capitalize?

Reply:
I only extend the default value.
![image](https://user-images.githubusercontent.com/42857405/109194354-f1a9cb80-775e-11eb-87c7-323813ae9096.png)